lib/pg_ltree/ltree.rb

```diff
@@ -96,7 +97,11 @@ def ltree_path
       #
       # @return [String] ltree previous value
       def ltree_path_was
-        public_send :"#{ltree_path_column}_was"
+        if active_record_5_1_or_above?
```

Comment:
I've move this to a class method that is initialized once. I was reluctant to place this in a `def initialize` since this is after all an ActiveRecord object we are mixing into.